create_assignments_from_csv_readme.md

```diff
@@ -2,6 +2,8 @@
 
 This script reads a CSV file containing the required fields to add unassigned or assigned assignments to a workforce project. Attachments can also be uploaded if the path to the file is specified in the CSV file.
 
+You can also use this script without the "xField" and "yField" parameters - the script will then geocode based on the "Location" field. Please note that geocoding consumes credits.
```

Comment:
consider re-phrasing to `... will then geocode the provided `location-field` to get the assignments location.`